pecl install protobuf3.11.0 error

env:
mac os: 10.14.4 (18E226)
php: 7.2.23

pecl install protobuf-3.11.0

error

```
cc ${wl}-flat_namespace ${wl}-undefined ${wl}suppress -o .libs/protobuf.so -bundle  .libs/array.o .libs/def.o .libs/encode_decode.o .libs/map.o .libs/message.o .libs/protobuf.o .libs/storage.o .libs/type_check.o .libs/upb.o .libs/utf8.o
duplicate symbol '_protobuf_globals' in:
    .libs/array.o
    .libs/def.o
duplicate symbol '_protobuf_globals' in:
    .libs/array.o
    .libs/encode_decode.o
duplicate symbol '_protobuf_globals' in:
    .libs/array.o
    .libs/map.o
duplicate symbol '_protobuf_globals' in:
    .libs/array.o
    .libs/message.o
duplicate symbol '_protobuf_globals' in:
    .libs/array.o
    .libs/protobuf.o
duplicate symbol '_protobuf_globals' in:
    .libs/array.o
    .libs/storage.o
duplicate symbol '_protobuf_globals' in:
    .libs/array.o
    .libs/type_check.o
ld: 7 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [protobuf.la] Error 1
ERROR: `make' failed
```

--- a/php/ext/google/protobuf/protobuf.c
+++ b/php/ext/google/protobuf/protobuf.c
@@ -39,6 +39,8 @@ static PHP_RSHUTDOWN_FUNCTION(protobuf);
 static PHP_MINIT_FUNCTION(protobuf);
 static PHP_MSHUTDOWN_FUNCTION(protobuf);
 
+ZEND_DECLARE_MODULE_GLOBALS(protobuf)
+
 // Global map from upb {msg,enum}defs to wrapper Descriptor/EnumDescriptor
 // instances.
 static HashTable* upb_def_to_php_obj_map;

--- a/php/ext/google/protobuf/protobuf.h
+++ b/php/ext/google/protobuf/protobuf.h
@@ -689,7 +689,7 @@ ZEND_BEGIN_MODULE_GLOBALS(protobuf)
   zend_bool keep_descriptor_pool_after_request;
 ZEND_END_MODULE_GLOBALS(protobuf)
 
-ZEND_DECLARE_MODULE_GLOBALS(protobuf)
+ZEND_EXTERN_MODULE_GLOBALS(protobuf)
 
 #ifdef ZTS
 #define PROTOBUF_G(v) TSRMG(protobuf_globals_id, zend_protobuf_globals *, v)

